ui/ui-frontend/projects/pastis/src/app/core/services/profile.service.ts

```diff
@@ -282,7 +282,7 @@ export class ProfileService implements OnDestroy {
   private updateControlSchema(archivalUnitProfile: ArchivalProfileUnit): ArchivalProfileUnit {
     try {
       const parsedControlSchema = JSON.parse(archivalUnitProfile.controlSchema);
-      const additionalProperties = JSON.parse(this.controlSchema.getValue())?.additionalProperties;
+      const additionalProperties = JSON.parse(this.controlSchema.getValue())?.additionalProperties ?? false;
```

Comment:
Je préfère la notation suivant, mais peu importe :
```suggestion
      const additionalProperties = !!JSON.parse(this.controlSchema.getValue())?.additionalProperties;
```